gulpfile.js

```diff
@@ -0,0 +1,249 @@
+var gulp = require("gulp"),
```

Comment:
There's quite a lot of boilerplate in all these SlamData projects (that is, the gulp files look almost the same). Would it make sense to factor this "skeleton" out into a gulp plugin, for example?